app/views/sessions/login.html.erb

```diff
@@ -2,21 +2,60 @@
 
   <h1>Log In</h1>
 
-  <p><i>Use your <a href="http://publiclab.org" target="_blank">PublicLab.org</a> username to log in. If you do not yet have one, <a href="http://publiclab.org/signup">sign up there</a> first!</i></p>
+  <p style="font-family:'Junction Light','lucida grande','lucida sans console',sans-serif;font-size: 15px;">
+    Login with your <a href="http://publiclab.org" target="_blank">PublicLab.org</a> username
+  </p>
+  <div class='hr-or' style="font-size: 10px;"></div>
+  <p style="font-family:'Junction Light','lucida grande','lucida sans console',sans-serif;margin-left: 40px;font-size: 15px;">
+    Create a <a href="http://publiclab.org/signup">Public Lab account</a>
+  </p>
 
   <form id="open_id_form" method="GET" action="/session/new">
 
     <%= hidden_field_tag :authenticity_token, form_authenticity_token -%>
-    <%= hidden_field_tag :back_to,@referer  -%>
-
-    <input tabindex="1" placeholder="username" id="username" name="open_id" type="text" /></p>
-    <input type="hidden" name="return_to" value="<%= params[:return_to] %>" />
-
-    <label class="checkbox inline" for="remember_me">Remember</label>
-    <input id="remember_me" name="remember_me" type="checkbox" value="1" />
-    <br>
-    <button tabindex="2" class="btn btn-large btn-primary" type="submit">Log in</button>
+    <%= hidden_field_tag :back_to, (params[:back_to] || request.env['PATH_INFO']) -%>
+    
+    <div>
+      <input type="hidden" name="return_to" value="<%= params[:back_to] || request.env['PATH_INFO'] %>" />
+    </div>
```

Comment:
They haven't been deleted, they were over-written by the new code

Reply:
Please see carefully, original line number 10 and 13 is modified to line 19 in new code. Is this necessary?
I think we don't need to modify this line.

Reply:
Yes it has been modified, but I didn't do that, they were in the latest pull. Is that not okay?